src/Reflection/AbstractFunction.php

```diff
@@ -91,13 +88,10 @@ abstract class AbstractFunction
     /**
      * Constructor
      *
-     * @param ReflectionFunctionAbstract $r
-     * @param null|string $namespace
-     * @param null|array $argv
      * @throws Exception\InvalidArgumentException
      * @throws Exception\RuntimeException
      */
-    public function __construct(ReflectionFunctionAbstract $r, $namespace = null, $argv = [])
+    public function __construct(ReflectionFunctionAbstract $r, ?string $namespace = null, ?array $argv = null)
```

Comment:
Signature change: default value `[]` -> `null`. Not sure why? I see default value in the property is `[]` so we can remove it, keep just array here, and remove the check `is_array` below.

Reply:
Totally makes sense. 👍🏻 Fixed.